src/utils/helpers/evaluateElements.ts

```diff
@@ -0,0 +1,89 @@
+import evaluateExpression, { isEvaluationExpression } from '@openmsupply/expression-evaluator'
+import config from '../../config.json'
+import {
+  EvaluatedElement,
+  ResponsesByCode,
+  User,
+  ApplicationDetails,
+  ElementForEvaluation,
+  EvaluationOptions,
+  EvaluatorNode,
+} from '../types'
+const graphQLEndpoint = config.serverGraphQL
+
+type PartialEvaluatedElement = Partial<EvaluatedElement>
+type EvaluationObject = {
+  responseObject?: ResponsesByCode
+  currentUser?: User | null
+  applicationData?: ApplicationDetails
+}
+
+type EvaluateElements = (
+  elements: ElementForEvaluation[],
+  evaluationOptions: EvaluationOptions,
+  objects: EvaluationObject
+) => Promise<PartialEvaluatedElement[]>
+
+const evaluationMapping: { [resultKey in keyof EvaluatedElement]: keyof ElementForEvaluation } = {
+  isEditable: 'isEditableExpression',
+  isRequired: 'isRequiredExpression',
+  isVisible: 'isVisibleExpression',
+  isValid: 'validationExpression',
+  defaultValue: 'defaultValueExpression',
+}
+
+export const evaluateElements: EvaluateElements = async (elements, evaluationOptions, objects) => {
+  const elementPromiseArray: Promise<PartialEvaluatedElement>[] = []
+  elements.forEach((element) => {
+    elementPromiseArray.push(evaluateSingleElement(element, evaluationOptions, objects))
+  })
+  return await Promise.all<PartialEvaluatedElement>(elementPromiseArray)
+}
+
+type EvaluatElement = (
```

Comment:
Spelling of "Evaluat**e**Element"

Reply:
9ea45719f4b73ff00128491aaf15cd68e23c1c04